cli/command/logs/follow.go

```diff
@@ -57,7 +58,10 @@ func runFollow(storageosCli *command.StorageOSCli, opt logOptions) error {
 		defer c.Close()
 		defer close(done)
 		for {
-			c.SetReadDeadline(time.Now().Add(wait))
+			if err := c.SetReadDeadline(time.Now().Add(wait)); err != nil {
+				log.Error("Failed to SetReadDeadline:", err)
```

Comment:
Should we log it like this?

Reply:
I think in general the CLI should only report fatal errors to screen, and the rest should be visible only when `--debug` is enabled.  In this case it looks like it can't continue so probably worth showing.